eZ/Publish/Core/Repository/PermissionsCriterionHandler.php

```diff
@@ -23,13 +24,27 @@
 class PermissionsCriterionHandler
 {
     /**
+     * @var \eZ\Publish\API\Repository\PermissionResolver
+     */
+    private $permissionResolver;
+
+    /**
+     * @var \eZ\Publish\Core\Repository\Helper\LimitationService
+     */
+    private $limitationService;
+
+    /**
      * Constructor.
      *
-     * @param \eZ\Publish\API\Repository\Repository $repository
+     * @param \eZ\Publish\API\Repository\PermissionResolver $permissionResolver
+     * @param \eZ\Publish\Core\Repository\Helper\LimitationService $limitationService
      */
-    public function __construct(RepositoryInterface $repository)
-    {
-        $this->repository = $repository;
+    public function __construct(
+        PermissionResolverInterface $permissionResolver,
+        LimitationService $limitationService
+    ) {
+        $this->PermissionResolver = $permissionResolver;
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/1720/commits/c3a60b9d14944b963ca3998021c5880dfba328bc